types/react-widgets/index.d.ts

```diff
@@ -8,6 +8,8 @@
 //                 Georg Steinmetz <https://github.com/georg94>
 //                 Troy Zarger <https://github.com/tzarger>
 //                 Siya Mzam  <https://github.com/vegtelenseg>
+//                 Cale Bergh <https://github.com/calebergh>
+//                 Ryan McKeel <https://github.com/rmckeel>
```

Comment:
Did you guys work together? :)

Reply:
@r3nya Yes, Cale and I pair coded / worked together on this patch.

Reply:
Ah, got it. Thank you!